packages/alice/lib/core/alice_logger.dart

```diff
@@ -2,77 +2,48 @@ import 'dart:io' show Process, ProcessResult;
 
 import 'package:alice/helper/operating_system.dart';
 import 'package:alice/model/alice_log.dart';
-import 'package:alice/utils/num_comparison.dart';
-import 'package:flutter/foundation.dart';
+import 'package:rxdart/rxdart.dart';
 
 /// Logger used to handle logs from application.
 class AliceLogger {
-  AliceLogger({int? maximumSize = 1000}) : _maximumSize = maximumSize;
+  /// Maximum logs size. If null, logs will be not rotated.
+  final int? maximumSize;
```

Comment:
Is there a difference between `null` and `0`?

Reply:
OK changed to 0, it makes sense to remove nullable if it doesn't make any difference.